backend/internal/service/handler/media/get_artist.go

```diff
@@ -0,0 +1,26 @@
+package media
+
+import (
+	"platnm/internal/models"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+func (h *Handler) GetMediaArtist(c *fiber.Ctx) error {
```

Comment:
just wondering, and ik you already put the effort into building this (🥲), but did you guys consider just joining the track/album table with the artist table to return the result in the existing endpoint that you're using to populate the page?

I think that would be a bit more efficient and save us some endpoint calls

this [feed PR](https://github.com/GenerateNU/platnm/pull/124) is an example (albeit a complex one) but it creates a custom struct with all the data needed, which is something that might benefit you guys here
(ex a PopulatedAlbum and PopulatedTrack type)